### ISSUE TITLE
Various a11y / UX improvements

### ISSUE THREAD

**Issue by jkva** [benetech/ImageShare#126]
## main page

* remove "logo" from alt text

* add search role to search form
	- restrict to search div

* wrap advanced button in h3

* add no feature collections placeholder

* reword "Jump" to "skip"

## search

* rebuild single result to resources and collection when applicable

* skip to main content when no results

* fix pluralisation

* h1 "search results"

* move reset button under search h2

* reword reset button to "Clear search parameters"

* add offscreen "Results" h2

* move resources/collections h2 to h3

* reword [x] per page to resources and collections

* implement pagination ul, named "resource pagination" / "collection pagination"

* move pagination section _below_ resources

* add h3 in pagination controls box

## resource page

* fallback got to search anchor

* see if there's a problem with tags not being rendered	

* aria-label "Files" to list ul

* fix bug with draft files being linked

* move download all files below heading

* download links labelledby heading

* frog dissection URI down error

## collections

* entire collection should be clickable

* collection list needs aria-label

* resource collection reference list needs aria-label

**Comment by jkva** [comment]
Implemented through various commits, among these 68c342cf7cdff536aa96511e5a48bc19ed3f807f

**Comment by sinabahram** [comment]
Opening this back up because markup errors are preventing me from verifying things like headings.

https://validator.w3.org/nu/?doc=http%3A%2F%2Fimgsdev.wpengine.com%2F%3Fpage%3Dsearch%26q%3Dphoto%26subject%3D%26type%3D%26acc%3D

**Comment by jkva** [comment]
Thanks for catching @sinabahram  - pushed a fix a minute ago.

**Comment by sinabahram** [comment]
Looping @jscholes  into this because those headings in search results are not showing up as headings, even though they report as headings when navigating by heading and when tabbed to, but not when just exploring with the virtual cursor. I fear this may be an issue of heading inside the link instead of heading outside the link, but the spec is fine with both, so I'm a bit baffled by this. James, can you please advise? Here's a link below directly to search results exhibiting the issue.

[search results for photo](http://imgsdev.wpengine.com/?page=search&q=photo&subject=&type=&acc=)

**Comment by sinabahram** [comment]
update: this is not happening in Firefox, so this may be a Chrome bug.

**Comment by sinabahram** [comment]
Another update, I let Aaron Leventhal at Google know about this behavior as well, in case he can figure something out here in Chrome.

**Comment by sinabahram** [comment]
Closing this because we're tracking heading issue in #127
